aries-site/src/examples/components/togglegroup/ToggleGroupViews.js

```diff
@@ -0,0 +1,81 @@
+import React, { useEffect, useState } from 'react';
+import {
+  Data,
+  DataFilters,
+  DataSearch,
+  DataSummary,
+  DataTable,
+  Pagination,
+  Toolbar,
+  ToggleGroup,
+} from 'grommet';
+import { Apps, Table, List as ListIcon } from 'grommet-icons';
+import { columns, fetchLaunches, formatData } from './utils';
+import { CardView } from './CardView';
+import { ListView } from './ListView';
+
+const toggleOptions = [
+  {
+    icon: <Table a11yTitle="Table view" />,
+    value: 'table',
+  },
+  {
+    icon: <ListIcon a11yTitle="List view" />,
+    value: 'list',
+  },
+  {
+    icon: <Apps a11yTitle="Card view" />,
+    value: 'card',
+  },
+];
+
+export const ToggleGroupViews = () => {
+  const [result, setResult] = useState({ data: [] });
+  // fetch launch data
+  useEffect(() => {
+    fetchLaunches('https://api.spacexdata.com/v4/launches/query').then(
+      response => {
+        setResult({
+          data: formatData(response.docs),
+        });
+      },
+    );
+  }, []);
+  const [value, setValue] = useState('table');
+  return (
+    <Data
+      data={result.data}
+      properties={{
+        name: { filter: false },
+        rocket: { label: 'Rocket' },
+        cores: { label: 'Cores' },
+        date_utc: { label: 'Date' },
+        success: { label: 'Success' },
+        failureAltitude: { label: 'Failure altitude' },
+      }}
+    >
+      <Toolbar>
+        <DataSearch />
+        <DataFilters layer />
+        <ToggleGroup
+          onToggle={e => {
+            if (e.value.length) setValue(e.value);
+          }}
+          value={value}
+          options={toggleOptions}
+        />
+      </Toolbar>
+      <DataSummary />
+      {value === 'table' && (
+        <DataTable
+          aria-describedby="spaceX"
```

Comment:
```suggestion
```
It doesn't look like any element has this id, should we remove?

Reply:
I can remove but it was throwing an error

Reply:
What error do you get?

Reply:
<img width="897" alt="Screen Shot 2024-06-17 at 3 51 19 PM" src="https://github.com/grommet/hpe-design-system/assets/42451602/042ed8ff-ae7b-4de4-8c20-dd7fe2591786">

Reply:
I can disable

Reply:
Maybe we can just add some text with the id that describes the datatable? Looks like that is the intent of the rule
https://github.com/grommet/eslint-plugin-grommet/blob/master/docs/rules/datatable-aria-describedby.md

Reply:
okay I can add

Reply:
done